### ISSUE TITLE
Add ability to connect to solution of external projects as internal

### ISSUE THREAD

**Issue by Zymlex** [intellij-rust/intellij-rust#6232]
Add ability to connect to solution of external projects as internal:
![изображение](https://user-images.githubusercontent.com/25817412/95600661-d54e1080-0a95-11eb-9f4c-495f6e5e099f.png)
(screen from CLion)

**Comment by Undin** [comment]
@Zymlex do I understand correctly, that you want to see several independent projects (i.e. they are not a part of the same Cargo workspace) located in different directories in the file system at the same time in Project View?

**Comment by Zymlex** [comment]
Yes, it is planned to use it in different solutions, and external projects will be refined in the process.
